.github/workflows/test-e2e.yaml

```diff
@@ -0,0 +1,154 @@
+# This file is used for tests that use Nx app containers.
+# For tests that don't require containers to be built first, see test.yaml
```

Comment:
```suggestion
# For tests that don't require containers to be built first, see test-code.yaml
```

I assume this was a leftover comment meant to be `test-code.yaml`, since `test.yaml` has been deleted?

Reply:
Thanks for catching these!